loader.mjs

```diff
@@ -53,7 +53,7 @@ export function transformSource(source, context, defaultTransformSource) {
       sourcefile: filename,
       sourcemap: 'both',
       loader: 'ts',
-      target: 'esnext',
+      target: `node${process.versions.node}`,
```

Comment:
Does esbuild supports semver on this? Or do we need to trunk it to the major version?

Reply:
it does support specific version https://esbuild.github.io/api/#target